core/src/main/java/org/p2p/core/utils/ListExtensions.kt

```diff
@@ -0,0 +1,5 @@
+package org.p2p.core.utils
+
+fun <T> merge(first: List<T>, second: List<T>): List<T> {
```

Comment:
-> StdExtensions

also, why not extension?

Reply:
function is more clear in code

Reply:
doubt that😁 
my first thought that it was some coroutine method and then i started looking some private method in presenters and was surprised that this merge only for lists
i suggest making it as extension, we can merge whatever we want, bind some context and sense to it

Reply:
extension on what can you please write
because for me it is more clear like Join function
not extension